tests/puppeteer/campaigns/functional/BO/catalog/categories/01_filterAndQuickEditCategory.js

```diff
@@ -0,0 +1,235 @@
+require('module-alias/register');
+// Using chai
+const {expect} = require('chai');
+const helper = require('@utils/helpers');
+const loginCommon = require('@commonTests/loginBO');
+const {Categories} = require('@data/demo/categories');
+// Importing pages
+const BOBasePage = require('@pages/BO/BObasePage');
+const LoginPage = require('@pages/BO/login');
+const DashboardPage = require('@pages/BO/dashboard');
+const CategoriesPage = require('@pages/BO/categories');
+
+let browser;
+let page;
+let numberOfCategories = 0;
+
+// Init objects needed
+const init = async function () {
+  return {
+    boBasePage: new BOBasePage(page),
+    loginPage: new LoginPage(page),
+    dashboardPage: new DashboardPage(page),
+    categoriesPage: new CategoriesPage(page),
+  };
+};
+
+// Filter And Quick Edit Categories
+describe('Filter And Quick Edit Categories', async () => {
+  // before and after functions
+  before(async function () {
+    browser = await helper.createBrowser();
+    page = await helper.newTab(browser);
+    this.pageObjects = await init();
+  });
+  after(async () => {
+    await helper.closeBrowser(browser);
+  });
+  // Login into BO and go to categories page
+  loginCommon.loginBO();
+  it('should go to "Catalog>Categories" page', async function () {
+    await this.pageObjects.boBasePage.goToSubMenu(
+      this.pageObjects.boBasePage.productsParentLink,
+      this.pageObjects.boBasePage.categoriesLink);
```

Comment:
Make sure that we know where the function line end (like that)
```suggestion
      this.pageObjects.boBasePage.categoriesLink
   );
```